<<<<<<<<<<<<<<<<<<<<<<< fife-9999-unbundle-libpng.patch >>>>>>>>>>>>>>>>>>>>>>>
Use local libpng instead of bundled one.

Patch by: Marcel Unbehaun, Azamat H. Hackimov, totycro, Mario Kicherer & Michal Plichta.

--- a/engine/core/video/image.h	(revision 3239)
+++ b/engine/core/video/image.h	(working copy)
@@ -27,7 +27,6 @@
 
 // 3rd party library includes
 #include <SDL.h>
-#include <png.h>
 
 // FIFE includes
 // These includes are split up in two parts, separated by one empty line

--- a/engine/core/video/image.cpp	(revision 3239)
+++ b/engine/core/video/image.cpp	(working copy)
@@ -25,6 +25,7 @@
 
 // 3rd party library includes
 #include <SDL.h>
+#include <png.h>
 
 // FIFE includes
 // These includes are split up in two parts, separated by one empty line

<<<<<<<<<<<<<<<<<<<<<<< fife-9999-unbundle-libpng.patch >>>>>>>>>>>>>>>>>>>>>>>
